server/api/test/index.js

```diff
@@ -46,7 +47,7 @@ describe( 'api', function() {
 	} );
 
 	let maybeIt = it;
-	if ( process.env.NODE_ENV !== 'desktop' ) {
```

Comment:
But if we are manually settings NODE_ENV to desktop, then we arent getting the massive perf boost of having all our deps in production mode.